style.css

```diff
@@ -0,0 +1,273 @@
+body{
+    margin: 0;
+    font-family: Roboto, sans-serif;
+}
+
+.main-container{
+    height:100vh;
+    width: 100vw;
+    display: grid;
+    grid-template-columns: 1fr 4fr;
+    grid-template-rows: 1fr 4fr;
+}
+
+.sidebar{
+    padding: 20px 20px 0px 30px;
+    background-color: #0284c7;
+    grid-row: 1/3;
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+}
+
+.sidebar > *{
+    color: white;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.dashboard-content{
+    align-items: center;
+    font-size: 24px;
+    font-weight: 800;
+    display: flex;
+    gap: 10px;
+}
+
+.sidebar img{
+    filter: invert(1);
+    height: 25px;
+}
+
+.dashboard-content img{
+    height:50px
```

Comment:
confirm if we explicitly need to define `width: auto`

Reply:
the initial value of width is set to `auto` so it doesn't need to be explicitly defined